src/configuration.ts

```diff
@@ -1,30 +1,26 @@
-import {Extractor, PullConfiguration, Regex, Rule, Sort} from './pr-collector/types'
+import {Extractor, PullConfiguration, Regex, Rule} from './pr-collector/types'
 
 export interface Configuration extends PullConfiguration {
-  max_tags_to_fetch: number
-  max_pull_requests: number
-  max_back_track_time_days: number
-  exclude_merge_branches: string[]
```

Comment:
why the removal of these?

Reply:
They are already inherited from `PullConfiguration`.